### DISCUSSION TITLE
在 VitePress 项目中使用 | OhMyLive2D

### DISCUSSION THREAD

**Discussion by giscus** [oh-my-live2d/oh-my-live2d#29] [Announcements]
# 在 VitePress 项目中使用 | OhMyLive2D

支持所有Live2D模型版本的Web组件, 快速为您的个人网站加入看板娘

https://oml2d.com/guide/vitepress.html

**Comment by hacxy** [comment]
不错喔~

**Comment by ATQQ** [comment]
Nice 👍🏻

**Comment by XingJi-love** [comment]
感谢，已点star

**Comment by webEngineerWong** [comment]
非常厉害我的vitepress 用了你的看板娘插件效果非常好。https://zubylon.tech/blog

**Comment by Peter-JXL** [comment]
想请教下如何调整看板娘的位置，或者让它不挡住页面元素

我用的 VitePress，我将其配置在左下角时，会挡住文档链接，点不了了

将其配置在右下角时，会挡住回到顶部按钮，按钮点不了了
